src/main/resources/api/service_openapi.yaml

```diff
@@ -440,41 +455,49 @@ components:
           items:
             type: string
 
-    JobControl:
+    JobReport:
       type: object
-      required: [jobid]
-      properties:
-        jobid:
-          type: string
-        pubsub:
-          type: object
-          required: [projectid, topicid]
-          properties:
-            projectid:
-              type: string
-            topicid:
-              type: string
-
-    JobModel:
-      type: object
-      required: [id, status, status_code]
+      required: [id, status, statusCode, resultURL]
       properties:
         id:
+          description: caller-provided unique identifier for the job
           type: string
         description:
+          description: caller-provided description of the job
           type: string
         status:
+          description: status of the job
           type: string
           enum: [RUNNING, SUCCEEDED, FAILED]
         statusCode:
-          description: HTTP code
+          description: >-
```

Comment:
I thought I read above that this was going away.

Reply:
I removed it from one internal method of JobService where all callers were ignoring it, but it still gets populated normally as part of building a JobReport